app/locale/en.coffee

```diff
@@ -3450,6 +3452,21 @@ module.exports = nativeDescription: "English", englishDescription: "English", tr
     final_arena: "Final Arena"
     package_options_and_guide: "Package Options & Implementation Guide"
     custom_tournament: "For a Custom Tournament"
+    you_win: "You can win"
+    great_prizes: "great prizes"
+    powered_by: "Powered by"
+    grand_prize: "Grand Prize"
+    scholarship: "Scholarship"
+    top_players_win: "Top three players in each division win: HyperX pro peripherals"
+    team_hyperx: "Team HyperX"
+    earn_more_gear: "you could earn even more gear!"
+    prize_footer1: "Awarded to the top performing player across all age groups."
+    prize_footer2: "Specific products and models subject to change."
+    prize_footer3: "Prizes will be awarded to players who reach the top of the leaderboard in the Finals arena."
+    prize_footer4_1: "Age verification necessary to be eligible.Some prizes are limited to US participants only."
+    coco_reserves: "CodeCombat reserves"
+    prize_footer4_2: "the right to determine in its sole discretion if a player qualifies and will receive a prize."
+    position: 'position'
```

Comment:
```suggestion
    place: "place"  # as in "1st place"
```
1. If I recall correctly, single quotes can work but sometimes get slightly funky with the copy-i18n-tags.js script.
2. "1st place" is more natural than "1st position", at least in en-US
3. With short ambiguous phrases like this, it's helpful to add a note to translators in the comment